covasim/webapp/index.html

```diff
@@ -36,199 +36,222 @@
-        <b-nav-item v-b-modal.about>About</b-nav-item>
+      <b-navbar-nav class="nav-head">
+        <b-nav-item :href="app.docs_url" target="_blank"><span class="nav-text">Documentation</span></b-nav-item>
+        <b-nav-item v-b-modal.about><span class="nav-text">About</span></b-nav-item>
       </b-navbar-nav>
       <b-navbar-nav class="ml-auto">
-        <b-nav-item :href="app.github_url" target="_blank">{{app.version}}</b-nav-item>
+        <b-nav-item :href="app.github_url" target="_blank"><span class="nav-text">{{app.version}}</span></b-nav-item>
       </b-navbar-nav>
     </b-navbar>
 
     <!-- About modal -->
     <b-modal id="about" ok-only ok-variant="secondary" ok-title="Close">
       <template v-slot:modal-title>About {{ app.title }}</template>
       <p>{{ app.title }} is an agent-based model that simulates the transmission of COVID-19 (novel coronavirus, SARS-CoV-2) in a population. Each individual in the model may pass through the following stages of COVID-19 infection: susceptible, exposed, infectious, and recovered (SEIR). The epidemiological parameter values are taken from literature.</p>
-      <p>Viral transmission occurs on a fixed contact network with undirected edges. Each day, infectious individuals expose susceptible individuals in neighboring nodes to possible infection. By default, the daily probability of infection, number of contacts, and mean duration of infectiousness roughly equates to R0 = 2.05 and a doubling time of 6-7 days. For more information, please visit the <a href="{{ app.github_url }}" target="_blank">GitHub</a> repository.</p>
-      <p>Note: Models are only as good as the values of the parameters put into them. This is a rapidly changing situation, and we will update content as relevant information is received.</p>
+      <p>Viral transmission occurs on a fixed contact network with undirected edges. Each day, infectious individuals expose susceptible individuals to possible infection. By default, the daily probability of infection, number of contacts, and mean duration of infectiousness roughly equates to R0 = 2.5 and a doubling time of 6-7 days. For more information, please visit the <a href="https://github.com/InstituteforDiseaseModeling/covasim" target="_blank">GitHub</a> repository.</p>
+      <p><b>Note:</b> Models are only as good as the values of the parameters put into them. This is a rapidly changing situation, and we will update content as relevant information is received.</p>
+      <p><b>Acknowledgements:</b> Covasim is currently under active development by IDM's COVID-19 Response Team. Model development is led by Daniel Klein, Cliff Kerr, Robyn Stuart, Romesh Abeysuriya, and Dina Mistry. UI design is led by Lauren George, William Chen, and Scott Ayers, inspired by a prototype from Brian Lovin. For the full list of contributors, please see our <a href="https://github.com/InstituteforDiseaseModeling/covasim/graphs/contributors" target="_blank">Contributors</a> page</a>.</p>
```

Comment:
Please use the {{app.github_url}} tokens.

Reply:
This isn't working for some reason, will add to issue: 
![image](https://user-images.githubusercontent.com/3239256/79918379-216ea100-83e1-11ea-91b1-15abcf956d62.png)